modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```

Comment:
but for those messages there are no possible ambiguities, are there?  i think 'VideoConstraintMessage' is too vague when, with things around o/a, we have 'constraints' being expressed from both sides.